data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,454 @@ mission "Unfettered returning home"
+		on kill
+			fail
+			set "unfettered: outcast"
+			dialog `As you destroy the Lightning Bug, along with Mera and her crew in it, you receive an angry hail from the Unfettered. "You completely disrespected the rules of a duel and killed one of our most prominent members! You will not be welcome here anymore!"`
+	npc
+		government "Hai (Unfettered)"
+		to spawn
+			has "unfettered: fled with the relic"
+		personality unconstrained nemesis launching heroic marked
+		ship "Sea Scorpion (Shredder)" "Hunter"
+	on visit
+		dialog "Either you have not defeated the Unfettered challenger in orbit yet, or you have broken the terms of the duel."
+	to complete
+		not "unfettered: fled with the relic"
+		# Needed to make sure the mission doesn't auto complete if no ships spawn.
+		has "unfettered: beaten the bugs"
+		has "flagship model: Anomalocaris"
+	on complete
+		clear "unfettered: beaten the bugs"
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an ancient Hai warship to beat an Unfettered challenge, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! I saw everything, and the data I got on the effects of these exotic weapons will be quite useful for my research. Was it hard for you?"`
+			choice
+				`	"It was fun."`
+					goto next
+				`	"This was easy!"`
+				`	"It was a challenge."`
+					goto challenge
+			`	"Don't get overconfident, Captain," she muses. "I am sure you know what happens to such pilots.`
+				goto next
+			label challenge
+			`	"Oh, so we almost got that ship after all? Maybe I should have participated in the fight after all.`
+			label next
+			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, right now, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us true Hai!"`
+			`	Mera then walks off as she begins reviewing data from your fight on a small device. For now, it seems that the Unfettered Hai have some sort of respect for you, so long as you have the Anomalocaris.`
+
+
+
+mission "Undeserved Anomalocaris"
+	minor
+	invisible
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "ship model: Anomalocaris"
+		"reputation: Hai (Unfettered)" <= -1000
+		not "Hai Legacy 2: active"
+	on offer
+		conversation
+			`You land on this Unfettered world in possession of the Anomalocaris, and, as you do, countless Unfettered head to your ship, each armed as they always are.`
+			`	Except, this time, they seem even more intimidating than usual... they must want the ship.`
+			`	One of them sends you a private message. "You do not deserve to fly this relic of old, and you missed your opportunity for a glorious battle to earn that right!"`
```

Comment:
```suggestion
			`	One of them sends you a private message. "You do not deserve to fly this relic of old, and you missed your opportunity for a glorious battle to earn that right!"`
```
This can display before the duel mission. What do you mean I missed the opportunity? What opportunity? What glorious battle?

Reply:
there's a to display here so it'd only show if you were offered the mission

Reply:
This mission (Undeserved Ano) offers if you're -1000 or more, and aren't in the middle of doing Hai Legacy 2. Which means, it can offer before Hai Legacy 2, and it can offer before Hai Legacy 3. The to display says this dialog will show so long as you *don't* have Hai Legacy 3 offered.

So technically, this mission, if I'm -1000 rep with the Unfettered, can show up on the other Unfettered worlds if I decide to land there first instead of finish the delivery, and they'll be talking about this possible duel that I never got told about yet. For example, if I failed Hai Legacy 2's delivery, but kept the ship, if I landed on a Uhai world, it will still show me the dialog saying I missed a duel, when I never got offered HL3 to begin with, since the to display is based on me *not* being offered it.

Reply:
TLDR it should be based on if you *have* had HL3 offered, not if you haven't.

Reply:
getting stuff the wrong way around does seem like smth I would do - fixed